definitions-tests/expose/trait.cue

```diff
@@ -0,0 +1,34 @@
+import (
+	"strconv"
```

Comment:
The trait & component CUE file can be read from `vela-templates/`.

Reply:
Now `definition_test` will read target definition `.cue` files from `vela-templates/`, and match the counterpart test case based on the same relative file path.